src/cares_wrap.cc

```diff
@@ -270,31 +285,43 @@ void ares_sockstate_cb(void* data,
 }
 
 
-Local<Array> HostentToAddresses(Environment* env, struct hostent* host) {
+Local<Array> HostentToAddresses(Environment* env,
+                                struct hostent* host,
+                                Local<Array> prev = Local<Array>()) {
```

Comment:
The only difference `prev` makes is that if it is present, the values are appended to it rather than to a new array, right? In that case, I’d say `appendTo` would be a better name than `prev`.